setup.py

```diff
@@ -55,7 +55,7 @@
     'packtivity==0.10.0',
     'pyzmq==16.0.2',
     'pyOpenSSL==17.5.0',  # FIXME remove once yadage-schemas solves deps.
-    'reana-commons>=0.5.0.dev20190213,<0.6.0',
+    'reana-commons[kubernetes]>=0.5.0.dev20190220,<0.6.0',
```

Comment:
Please also upgrade package version to `0.5.0.dev20190220` so that we have the same version as in `r-commons`.